lib/nexmo_developer/app/presenters/head.rb

```diff
@@ -1,66 +1,96 @@
 class Head
-  attr_reader :items
+  class Description
+    def initialize(config:, frontmatter:)
+      @config      = config
+      @frontmatter = frontmatter
+    end
 
-  def initialize(items: nil)
-    @items = items
+    def description
+      @description ||= from_frontmatter || @config.fetch('description') do
+        raise "You must provide a 'description' parameter in header_meta.yml"
+      end
+    end
+
+    def from_frontmatter
+      @frontmatter && (@frontmatter['meta_description'] || @frontmatter['description'])
```

Comment:
Should we use kebab-case here for `meta_description` to match the rest of the parameters in the header meta YAML file?

Reply:
we can, but it wouldn't be backwards compatible :/, so I left it as it was

Reply:
Good point, yes let's leave it then